src/Moq/SetupCollection.cs

```diff
@@ -86,7 +86,7 @@ public Setup[] ToArrayLive(Func<Setup, bool> predicate)
 			// The following verification logic will remember each processed setup so that duplicate setups
 			// (that is, setups overridden by later setups with an equivalent expression) can be detected.
 			// To speed up duplicate detection, they are partitioned according to the method they target.
```

Comment:
That partitioning might no longer be as crucial as before, since the first equality check `Invocation.Equals` performs is for the `MethodInfo`s to be equal.